### PR TITLE
added changelog for react native

### PR DESCRIPTION
**Pull request by matiasholst**

### CODE REVIEW

**Comment by matiasholst** [comment]
@fasaMaps please look through this PR

File: docs/08-Changelogs/react-native.mdx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<!--
+### Added        for new features.
+### Changed      for changes in existing functionality.
+### Deprecated   for soon-to-be removed features.
+### Removed      for now removed features.
+### Fixed        for any bug fixes.
+### Security     in case of vulnerabilities.
+-->
+
+## [1.0.2] 2023-09-04
```
Comment (timMikkelsen) [inline docs/08-Changelogs/react-native.mdx:20]: Wrong version name + date

**Review by fasaMaps — APPROVED** [review]

**Comment by fasaMaps** [comment]
what Tim mentioned rest looks fine from POV :)

File: docs/08-Changelogs/react-native.mdx (hunk 1 above)
Comment (matiasholst) [reply docs/08-Changelogs/react-native.mdx]: nice catch

**Review by timMikkelsen — APPROVED** [review]

**Pull request merged**